web/src/beta/components/Button/README.md

```diff
@@ -0,0 +1,9 @@
+# Button
```

Comment:
This file isn't necessary any more. Please delete. Its very very old

Reply:
Please delete all of these files

Reply:
done